cli/command/plugin/list.go

```diff
@@ -17,6 +21,12 @@ type listOptions struct {
 	filter  opts.FilterOpt
 }
 
+type byName []*types.Plugin
```

Comment:
Can we use `sort.Slice` instead of defining a type for this?

Reply:
Thank you for the review. I will update the PR to use `sort.Slice`